spec/lex.dd

```diff
@@ -542,7 +547,8 @@ $(H2 $(LNAME2 escape_sequences, Escape Sequences))
         $(D \U0001F603) represents the Unicode character U+1F603 (SMILING FACE
         WITH OPEN MOUTH).)
     $(TROW $(D \)$(I name), Named character entity from the HTML5
-        specification. See $(GLINK2 entity, NamedCharacterEntity).)
+        specification. These names begin with $(I &) and end with $(I ;), e.g., $D(&euro;).
+        See $(GLINK2 entity, NamedCharacterEntity).)
```

Comment:
You may need to escape the `&`, there seem to be some latex errors coming from those.

Reply:
How do I escape &, I don't even know what markup language is being used; I've just tried to mimic what's there.

Reply:
The language is DDOC (see https://dlang.org/spec/ddoc). The escape for `&` is `&amp`.

Reply:
Or try `CODE_AMP` in place of `$(D &)`

Reply:
There is no correct answer here - this is ddoc's fatal design flaw. `&amp;` works if you are targeting HTML but is wrong for other targets like latex. The best you can do is whatever macro is defined in teh `.ddoc` file on your particular project.

Here it is `$(CODE_AMP)`.